install/tools/ipa-replica-conncheck

```diff
@@ -384,11 +442,8 @@ def main():
         # create listeners
         print_info("Start listening on required ports for remote master check")
 
-        for port in required_ports:
-            root_logger.debug("Start listening on port %d (%s)" % (port.port, port.description))
-            responder = PortResponder(port.port, port.port_type)
-            responder.start()
-            RESPONDERS.append(responder)
+        RESPONDER = PortResponder(required_ports)
```

Comment:
The design should have Implementation section mentioning why it is done in this particular way (i.e. explain why we have one thread vs. other options).